hist/hist/src/TMultiGraph.cxx

```diff
@@ -1095,8 +1095,15 @@ TH1F *TMultiGraph::GetHistogram()
    double dy = 0.05*(rwymax-rwymin);
    rwxmin = rwxmin - dx;
    rwxmax = rwxmax + dx;
-   rwymin = rwymin - dy;
-   rwymax = rwymax + dy;
+   if (gPad->GetLogy()) {
```

Comment:
Is this correction for ymin/ymax in case of logY scale used also in the case of `TGraph::GetHistogram()` ?
If this is different , why ? 
Is there also a macro/tutorial showing this improvement ?

Reply:
I have check TGraph is fine.
I added a test ... can you check it is done the correct way ?